CHANGELOG.md

```diff
@@ -6,13 +6,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
+- Models:
+    - `meeting` - for OSF Meetings
+- Adapters:
+    - `meeting` - in private namespace
+- Serializers:
+    - `meeting`
 - Routes:
     - `meetings` - parent route for meetings
         - `meetings.index` - meetings landing page
 - Components:
     - `get-started-button` - a button that takes you to the '/register' page.
     - `search-bar` - a search bar component that takes you to the search page.
     - `new-home/-components/hero-banner` - a banner to be used on the logged-out homepage.
+    - `meetings/index/components/meetings-list` - meetings list for the meetings index page
```

Comment:
Maybe also add `x-header` to this list of newly added components?

Reply:
👇